specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/bigDataPool.json

```
@@ -433,7 +433,12 @@
         "cacheSize": {
           "type": "integer",
           "format": "int32",
-          "description": "The cache size"
+          "description": "The cache size",
+          "x-ms-mutability": [
+            "create",
+            "read",
+            "update"
+          ]
```

This change only adds verbosity -- it is equivalent to omitting "x-ms-mutability".
```suggestion
          "description": "The cache size"
```